### PR TITLE
modified Marr post

### PR DESCRIPTION
**Pull request by rawskijon**

### CODE REVIEW

**Comment by thomas--graf** [comment]
I'm currently in Amsterdam, so it will probably take a few days before I merge this in. You can keep revising in the meantime if necessary.

**Comment by rawskijon** [comment]
Sorry for the pull request mess, I didn't understand at first that I could just keep one draft pull open while updating my fork.

File: content/Discussions/2019-12-31-Rawski_Marr.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+Personally, I tend to think of them as lenses.
+Each "level" is more or less refined, allowing different properties to emerge.
+For a delightful perspective, check out minutes 5-12 of [this '60s BBC episode](https://www.dailymotion.com/video/x20ohne) (with soundtrack by Pink Floyd!).    
+
+This is a very general perspective on explanation in information processing.
+But I think the only reason linguists pay any attention to Marr is because he spends two paragraphs on how "Chomsky's (1965) theory of transformational grammar is a true computational theory in the sense defined earlier".
+He places Chomsky's competence/performance distinction as the difference between the "computational theory" and "representation/algorithmic" levels.
+He even talks about trace theory a la Chomsky \& Lasnik showing "some of the rather ad hoc restrictions that form part of the computational theory may be consequences of weaknesses in the computational power that is available for implementing syntactical decoding".
+Linguists adapted this endorsement into the following argument (see above meme):
+
```
Comment (thomas--graf) [inline content/Discussions/2019-12-31-Rawski_Marr.md]: Is this actually a quote? If it isn't, typeset it differently. E.g.

```
(@folkmarr) **Linguists' take on Marr** (wow, that's one important apostrophe)  
Marr distinguishes...
```

File: content/Discussions/2019-12-31-Rawski_Marr.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+This makes a bit more sense under the representation-as-process view. However, I have an even harder time trying to fit modern linguistics into one level here. 
+
+![more levels]({static}/img/Rawski/Marr/7Lvls.jpg){ width=60% }
+
+In his [afterward to the re-release of the book](http://cbcl.mit.edu/publications/ps/marr-vision-afterword.pdf), Tommaso Poggio is critical of level independence.
+He recounts the gritty origin story of the Three Levels: while they were colleagues, Marr adapted the framework of @reichardt1976, which distinguished behavioral, algorithmic, and implementational levels (Marr swapped out 'computation' for behavior).
+Poggio notes that "one key aspect of the original argument disappeared in the process".
+What was it?
+"It is necessary to study nervous systems at all levels simultaneously".
+Great advice.
```
Comment (thomas--graf) [inline content/Discussions/2019-12-31-Rawski_Marr.md]: There might be trailing spaces here, there's a sudden linebreak in the typeset article.

File: content/Discussions/2019-12-31-Rawski_Marr.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+title: >-
+    Overappreciated Arguments: Marr's Three Levels
+authors:
+    - Jon Rawski
+date: 2019-12-31
+bibliography: rawski.bib
+tags:
+    - neuroscience
+    - representations
+    - marr
```
Comment (thomas--graf) [inline content/Discussions/2019-12-31-Rawski_Marr.md]: `- Marr`

**Comment by thomas--graf** [comment]
I spotted three very minor issues, check the comments in the code view. Once those are fixed, I'll merge it into `staging` and then push it out to `master` whenever you'd like (Sunday?). I might add a very short part at the top to say that this is a guest post and the Outdex would be happy to publish more of those.

**Comment by rawskijon** [comment]
Alright, I fixed those up, and some other nitpicks I noticed. I updated the date for Sunday, but whenever you'd like to merge is fine. And add any preface you'd like! Thanks for bearing with me as I navigate this process *insert quote on benefits of active learning*

**Pull request merged**